docs/sources/get-started/install/linux.md

```diff
@@ -78,6 +78,17 @@ sslcacert=/etc/pki/tls/certs/ca-bundle.crt' | sudo tee /etc/yum.repos.d/grafana.
    ```
    {{< /code >}}
 
+1. Optional: Edit the run configuration to enable the {{< param "PRODUCT_NAME" >}} UI
+
+   ```bash
+   sed -i -e 's/CUSTOM_ARGS=""/CUSTOM_ARGS="--server.http.listen-addr=0.0.0.0:12345"/' /etc/default/alloy
+   ```
+
+The {{< param "PRODUCT_NAME" >}} UI is a web application that runs on `http://localhost:12345/` and allows for
+[debugging your configuration] and viewing it in a visual format. Enabling this UI is recommended
+for beginners and is not necessary for network isolated, headless, or production installs. It also enables hot-reloading
+of the {{< param "PRODUCT_NAME" >}} configuration without restarting the system service.
```

Comment:
I suggest we replace the word optional with "For beginners/Alloy tutorial users". It's highly likely that people may skip over when they see the word optional and this step is important for beginners and Alloy tutorial users. 

I also made some grammatical changes to this section and have pasted it below:

For beginners/Alloy tutorial users: Edit the run configuration to enable the {{< param "PRODUCT_NAME" >}} UI

   ```bash
   sed -i -e 's/CUSTOM_ARGS=""/CUSTOM_ARGS="--server.http.listen-addr=0.0.0.0:12345"/' /etc/default/alloy
   ```

The {{< param "PRODUCT_NAME" >}} UI is a web application that runs on `http://localhost:12345/` and allows for
[debugging your configuration] and viewing it in a visual format. Enabling this UI is recommended
for beginners and unnecessary for network-isolated, headless, or production installs. It also enables hot-reloading
of the {{< param "PRODUCT_NAME" >}} configuration without restarting the system service.